samples/react-tabacordion/src/webparts/tabAccordion/components/CAccordion.tsx

```diff
@@ -0,0 +1,118 @@
+
+
+
+import * as React from 'react';
+import styles from './CAccordion.module.scss';
+import { ICAccordionProps } from './ICAccordionProps';
+import { DisplayMode, Version } from '@microsoft/sp-core-library';
+import { Editor } from '@tinymce/tinymce-react';
+
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemTitle,
+  AccordionItemBody,
+} from './utilities/Accordion/index';
+
+export default class CAccordion extends React.Component<ICAccordionProps, {}> {
+  public handleEditorChange = (e) => {
+    var id = e.target.id.split("-editor-")[1];
+    //Save the content in properties
+  
+    this.props.tabs[id].Content = e.target.getContent();
+  }
+  public render(): React.ReactElement<ICAccordionProps> {
+    if(this.props.displayMode === DisplayMode.Edit)  
+     {
+      return (
+        <div>                 
+         <div className={styles.webpartheader}>
+            <div className={styles.webparttitle}>
+              <span role="heading" aria-level={2}>{this.props.title}</span>
+            </div>
+         </div>
+        <Accordion className={styles.accordion} aria-live="polite" accordion={this.props.accordion}>
+          {           
+            this.props.tabs.map((tab: any, index: number) => {
+            return (
+             <AccordionItem key={"tab" + index} className={styles.accordion__item} aria-controls={this.props.guid + '-title-' + index} id={"tab" + index} >
+                  <AccordionItemTitle className={styles.accordion__title} id={this.props.guid + '-title-' + index}>
+                  <div className={styles.accordion__arrow} role="presentation" />
+                      <div className={styles["positionRelative"]}>                      
+                         {tab.Title}                          
+                      </div>
+                  </AccordionItemTitle>
+                  <AccordionItemBody className={styles.accordion__body} hideBodyClassName={styles["accordionBodyHidden"]} >                    
+                  <Editor
+                          id={this.props.guid + '-editor-' + index}                        
+                          value={tab.Content}
+                          init={{                          
+                            content_style: "a {color:rgb(0,120,212) !important}",
+                            plugins: 'link image table lists pageembed',
+                            menubar: 'edit insert format table lists view',  // skip file
```

Comment:
It would be awesome if you'd use localization instead of static strings. I think that people will most likely want to use your sample as is, and community contributors are most likely to want to add more languages

Reply:
these are more options or settings for the tiny-mce. I don't think this can be localized.